README.md

```diff
@@ -44,31 +43,30 @@ dependencies {
 
 ### Registering module
 
-To use JTS geometry datatypes with Jackson, you will first need to register the module first (same as
-with all Jackson datatype modules):
+To use JTS geometry datatypes with Jackson, you will first need to register the module first (same as with all Jackson
+datatype modules):
 
 ```java
-ObjectMapper mapper = new ObjectMapper();
-mapper.registerModule(new JtsModule());
+ObjectMapper mapper=new ObjectMapper();
+        mapper.registerModule(new JtsModule());
```

Comment:
I think you don't have to change this? the old variant (`ObjectMapper mapper = new ObjectMapper();`) looks fine to me

(maybe this was an automated change?)

Reply:
Indeed. Ill revert the automatic changes to get the diff more precise